code/server/models/User.js

```diff
@@ -57,6 +57,14 @@ var getUser = function(username) {
     });
 }
 
+var getUsersById = function(ids) {
```

Comment:
@MattMcMurray well....it kindof uses an array of ids and returns multiple values